pkg/secrethub/http/encoding.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // Errors
 var (
-	ErrWrongContentType = errClient.Code("wrong_content_type").Error("server returned wrong content type in header")
+	ErrWrongContentType = errHTTP.Code("wrong_content_type").Error("server returned wrong content type in header")
```

Comment:
Question, why don't we call this `errHTTP` variable simply `errors`? Not for now, I know, but it just popped into my head.

Reply:
`errors` collides with the package named `errors`.